server/cmwell-data-tools/src/main/resources/reference.conf

```diff
@@ -1,17 +1,21 @@
-akka {
-  log-dead-letters = 2
-  log-dead-letters-during-shutdown = on
-  #  log-config-on-start = on
-  #  jvm-exit-on-fatal-error = on
+data-tools {
+  akka {
+    log-dead-letters = 2
```

Comment:
IMHO, the akka logging parameters aren't data-tools specific. You can leave it to the default ones configured by the main application.